src/platform/plugins/private/presentation_panel/public/panel_component/panel_header/presentation_panel_actions_simple_wrapper.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React, { ReactElement } from 'react';
+import { useBatchedOptionalPublishingSubjects } from '@kbn/presentation-publishing';
+import { css } from '@emotion/react';
+import { DefaultPresentationPanelApi } from '../types';
+import { useHoverActionStyles } from './use_hover_actions_styles';
+
+const actionsStyles = css({
+  justifyContent: 'right',
+  '.embPanel__hoverActionsAnchor &': {
+    '&.embPanel__hoverActions': { visibility: 'visible !important' as 'visible' },
+  },
+});
+
+export const PresentationPanelActionsSimpleWrapper = ({
```

Comment:
I still can't help but wonder if there's a way to further consolidate these without a separate `Wrapper` and `SimpleWrapper` component.